cmd/config/subcommand/sandbox/sandbox_config.go

```diff
@@ -0,0 +1,14 @@
+package sandbox
+
+//go:generate pflags Config --default-var DefaultConfig
+var (
+	DefaultConfig = &Config{
+		Debug: false,
+	}
+)
+
+// Config
+type Config struct {
+	Debug      bool   `json:"debug" pflag:", Enable debugging"`
+	SnacksRepo string `json:"flytesnacks-path" pflag:", Path of your flytesnacks repository"`
```

Comment:
Avoid hyphens in commands line flags . to be consistent with others use camel case

Reply:
i used `flytesnacks`